src/solvers/extracter.jl

```diff
@@ -0,0 +1,17 @@
+import ConstructionBase: setproperties
+
+function extracter(problem, region_iterator; sweep, trunc=(;), kws...)
```

Comment:
If we don't change this to `extract` as discussed elsewhere, which should change this to `extractor` since `extracter` isn't the correct spelling.